apps/fluent-tester/src/E2E/BootPage/pages/BootTestPage.win.ts

```diff
@@ -1,19 +1,17 @@
-import {
-  HOMEPAGE_BUTTON_BUTTON,
-  HOMEPAGE_CALLOUT_BUTTON,
-  HOMEPAGE_CHECKBOX_BUTTON,
-  HOMEPAGE_FOCUSTRAPZONE_BUTTON,
-  HOMEPAGE_LINK_BUTTON,
-  HOMEPAGE_PERSONA_BUTTON,
-  HOMEPAGE_PERSONACOIN_BUTTON,
-  HOMEPAGE_PRESSABLE_BUTTON,
-  HOMEPAGE_RADIOGROUP_BUTTON,
-  HOMEPAGE_SEPARATOR_BUTTON,
-  HOMEPAGE_SVG_BUTTON,
-  HOMEPAGE_TEXT_BUTTON,
-  HOMEPAGE_THEME_BUTTON
-} from '../../RNTester/Consts';
-import { By } from './BasePage';
+import { HOMEPAGE_CHECKBOX_BUTTON } from '../../../RNTester/TestComponents/Checkbox/consts';
+import { HOMEPAGE_BUTTON_BUTTON } from '../../../RNTester/TestComponents/Button/consts';
+import { HOMEPAGE_CALLOUT_BUTTON } from '../../../RNTester/TestComponents/Callout/consts';
```

Comment:
I think you can re-export these from index file under RNTester to shorten the relative paths down to `'../../../RNTester'`. I think there are actually ways to get out of relative path hell using config options in the bundler to resolve paths to some alias. I'll file an issue on it.

Reply:
#221 😊

Reply:
I actually tried exporting from index files, but it was breaking the E2E testing with an error. I will try again, maybe I was doing it wrong. Will update this thread once I do this

Reply:
Post the error here, may be indicative of a larger problem

Reply:
Perhaps I'm implementing it incorrectly, but here's what I did (just testing for the Button test component right now). 

In the index file under RNTester, I added:
```export { HOMEPAGE_BUTTON_BUTTON, BUTTON_TESTPAGE } from './TestComponents/Button/consts';```

Then, in ButtonTestPage.win.ts and BootTestPage.win.ts (the two files that import these constants), I changed the path to:
```import { HOMEPAGE_BUTTON_BUTTON } from '../../../RNTester';```

When running the E2E tests, I get this error:
```Error: Cannot use import statement outside a module```

I'll look into this more, but do you see why this is happening?

Reply:
A shot in the dark, but does adding /index to the end change anything?

Reply:
could also be a transpilation thing with babel not transforming things correctly. I don't know a lot about that. @JasonVMo any ideas?

Reply:
Nope, adding /index doesn't change anything

Reply:
Here's a screenshot for further information.
![image](https://user-images.githubusercontent.com/32719011/82497435-16bc3000-9aa3-11ea-92a1-7c43dbdf36af.png)

Reply:
So I exported the consts in each test component's index file, and then imported them in the RNTester\TestComponents\index file, and that worked correctly.

But when I try to import those exact constants from the same location, but in this file (E2E/pages), it gives me the above error. Not sure why

Reply:
Do any imports work in this file? CommonJS and ES6 use different import/export syntax (it's a whole crazy thing).

Reply:
This works:
```import { HOMEPAGE_CHECKBOX_BUTTON } from '../../../RNTester/TestComponents/Checkbox/consts';```

But not this:
```import { HOMEPAGE_CHECKBOX_BUTTON } from '../../../RNTester/TestComponents/Checkbox';```

Even though I export it from 'Checkbox/index', and I use that exact import statement in the TestComponents\index file and it works